metrotranscom/doorway
# Add uic repo to doorway

This PR adds a copy of uic into Doorway and treats it like another service within the monorepo. I renamed it from `@bloom-housing/ui-components` to `@bloom-housing/doorway-ui-components` because there were some weird interactions happening. We need to do this to be able to reuse bloom components and just restyle.

I only updated one dependency to pull from `@bloom-housing/doorway-ui-components` which is the header, so you can test on that. If I were to change all dependencies this CL would balloon in scope greatly (and I'm not 100% sure that we want to switch it out yet). 

I also updated with a github action with this service's tests, and (controversially) do not lint this dir as there are lots of lint errors that I cannot fix as a single human. I think can alternatively push it with lint errors which lets these errors slide but will not let new ones.

To review:
* skip pretty much everything in the doorway-uic repo, IMO, given that it is 99% a copypaste job
* look at other dependencies

To test:
* `yarn` and start local sites to make sure that it works as normal (yarn will now also install dependencies for uic)
* with a local public site up, try changing some css within languagenav.scss within doorway-uic to see if it changes on the local version of the public site live.
